### PR TITLE
Add Vercel hosting guide (#543)

### PR DESCRIPTION
**Pull request by roj1512**
Co-authored-by: jgcarrillo <jgcarrillo@users.noreply.github.com>
Co-authored-by: Roj <rojserbest@gmail.com>
Co-authored-by: Ciki Momogi <git.fytyt@aleeas.com>
Co-authored-by: KnorpelSenf <shtrog@gmail.com>

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
@jgcarrillo now it's time to translate. Are you still interested in helping out with the Spanish version? If so, I'll grant you write access to this repo so that you can directly push to this branch.

**Comment by KnorpelSenf** [comment]
> > @jgcarrillo now it's time to translate. Are you still interested in helping out with the Spanish version? If so, I'll grant you write access to this repo so that you can directly push to this branch.
> 
> Of course! hope to do better this time since spanish is my mother language 😄 so assign it to me without problem!

Done! No more forking necessary.

Please check out the translation guide once before you start :)

/cc @habemuscode

File: site/docs/hosting/vercel.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import { Bot, webhookCallback } from "grammy";
+
+const bot = new Bot(process.env.BOT_TOKEN);
+
+export default webhookCallback(bot, "http");
+```
+
+## In Vercel's Dashboard
+
+Assuming that you already have a Vercel account your GitHub is connected to it, add a new project and select your bot's repository.
```
Comment (CikiMomogi) [inline site/docs/hosting/vercel.md:80]: Is it grammatically correct? It sounds weird to me :thinking:  
```suggestion
Assuming that you already have a Vercel account where your GitHub is connected to it, add a new project and select your bot's repository.
```

Comment (KnorpelSenf) [reply site/docs/hosting/vercel.md:80]: That's still not correct, this is what's meant here:
```suggestion
Assuming that you already have a Vercel account your GitHub is connected to, add a new project and select your bot's repository.
```
But perhaps that's better?
```suggestion
Assuming that you already have a Vercel account which is connected to GitHub, add a new project and select your bot's repository.
```

**Comment by habemuscode** [comment]
Spanish done

**Review by KnorpelSenf — APPROVED** [review]
Excellent!

**Pull request merged**